CHANGELOG.md

```diff
@@ -35,31 +34,40 @@ For example:
 @import "node_modules/govuk-frontend/govuk/components/header";
 ```
 
+Find out more about [importing CSS](https://frontend.design-system.service.gov.uk/importing-css-assets-and-javascript/#css).
+
 [Pull request #1804: Allow components to be imported without dependencies](https://github.com/alphagov/govuk-frontend/pull/1804).
 
+#### Back links and breadcrumbs are easier to select
+
+The [back link](https://design-system.service.gov.uk/components/back-link/) and [breadcrumbs](https://design-system.service.gov.uk/components/breadcrumbs/) components are now bigger, so they're easier to select if you're using a touch screen.
+
+The design of the back link component is now more consistent with the breadcrumb component. 
 
-### Deprecations
+[Pull request #1753: Make back link arrow consistent with breadcrumb component](https://github.com/alphagov/govuk-frontend/pull/1753).
 
-#### Importing files from the core or overrides layers
+### Deprecated features
 
-If you import specific files from the core or overrides layers, you should import the `base` file first:
+#### Importing from the `core` and `overrides` layers
```

Comment:
Might be fine, but if you were scanning the headings it does sound like we're deprecating being able to import from the core and override layers entirely.

Reply:
Good point - I'll add 'without `base`' if that sounds ok?